src/indev/lv_indev.h

```diff
@@ -285,6 +285,15 @@ lv_dir_t lv_indev_get_gesture_dir(const lv_indev_t * indev);
  */
 uint32_t lv_indev_get_key(const lv_indev_t * indev);
 
+
+/**
+ * Get the counter for consecutive clicks within a short distance and time.
+ * The counter is updated before LV_EVENT_SHORT_CLICKED is fired.
+ * @param indev pointer to an input device
+ * @return 0 based short click streak counter
```

Comment:
```suggestion
 * @param indev     pointer to an input device
 * @return          based short click streak counter
```

I suggest making it 1 based because it seems more intuitive.